server/routes/api/documents.test.ts

```diff
@@ -1432,12 +1433,76 @@ describe("#documents.archived", () => {
     expect(body.data.length).toEqual(0);
   });
 
+  it("should require member", async () => {
```

Comment:
Fixes a bug where viewers could load archived docs via API but not see them in the UI